build_projects/dotnet-host-build/CompileTargets.cs

```diff
@@ -37,14 +39,11 @@ public class CompileTargets
             { "debian.8-x64", "debian.8-x64" }
         };
 
-
-        private static string DotnetHostBaseName => $"dotnet{Constants.ExeSuffix}";
-        private static string DotnetHostFxrBaseName => $"{Constants.DynamicLibPrefix}hostfxr{Constants.DynamicLibSuffix}";
-        private static string HostPolicyBaseName => $"{Constants.DynamicLibPrefix}hostpolicy{Constants.DynamicLibSuffix}";
-
         [Target(nameof(PrepareTargets.Init), 
             nameof(CompileCoreHost), 
-            nameof(PackagePkgProjects))]
+            nameof(PackagePkgProjects),
+            nameof(RestoreLockedCoreHost),
```

Comment:
I thought you and Senthil said last time that RestoreLockedCoreHost was supposed to be in dotnet-cli-build?  [https://github.com/dotnet/cli/pull/3003#discussion_r62949648](https://github.com/dotnet/cli/pull/3003#discussion_r62949648)